php.ini-development

```diff
@@ -996,8 +996,19 @@ cli_server.color = On
 ;intl.use_exceptions = 0
 
 [sqlite3]
+; Directory pointing to SQLite3 extensions
+; http://php.net/manual/en/sqlite3.loadextension.php
```

Comment:
```suggestion
; https://php.net/manual/en/sqlite3.loadextension.php
```

Reply:
I'm not against it but all the other php.net URLs in the php.ini are http only, so I followed, if I proceed, it would be the only one to be in https. Not sure if that's ok?

Reply:
Makes no sense to switch to `https` one at a time; instead we should change all the URLs after all of php.net has switched to HTTPS.